### PR TITLE
Kubernetes IoT docs

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

**Comment by lulf** [comment]
@dejanb can you rebase this ?

File: documentation/modules/proc-iot-api.adoc [hunk 1]
```diff
@@ -22,13 +29,39 @@ mvn package -am
...
 ----
+ifeval::["{cmdcli}" == "kubectl"]
+{cmdcli} -n myapp get addressspace iot -o 'jsonpath={.status.caCert}' | base64 --decode > target/config/hono-demo-certs-jar/tls.crt
+endif::[]
+ifeval::["{cmdcli}" == "oc"]
 {cmdcli} -n myapp get addressspace iot -o jsonpath={.status.endpointStatuses[?\(@.name==\'messaging\'\)].cert} | base64 --decode > target/config/hono-demo-certs-jar/tls.crt
+endif::[]
+----
+
+. Export messaging endpoint host and port
```
Comment (jenmalloy) [inline documentation/modules/proc-iot-api.adoc]:
```suggestion
. Export messaging endpoint host and port:
```

File: documentation/modules/proc-iot-api.adoc [hunk 2]
```diff
@@ -22,13 +29,39 @@ mvn package -am
...
+[options="nowrap",subs="attributes"]
+----
+export MESSAGING_HOST=$({cmdcli} -n myapp get addressspace iot -o jsonpath={.status.endpointStatuses[?\(@.name==\'messaging\'\)].externalHost})
+export MESSAGING_PORT=443
 ----
+ifeval::["{cmdcli}" == "kubectl"]
++
+[NOTE]
+====
+If you're running Kubeneretes in development environment without proper load balancer, you need to use IP address of your local cluster and the port of the appropriate service, for example:
```
Comment (jenmalloy) [inline documentation/modules/proc-iot-api.adoc]:
```suggestion
If you are running Kubernetes in a development environment without a proper load balancer, you need to use the IP address of your local cluster and the port of the appropriate service; for example:
```

File: documentation/modules/proc-iot-installing-services.adoc [hunk 3]
```diff
@@ -13,11 +13,25 @@
 ----
 {cmdcli} apply -f install/preview-bundles/iot
 ----
+ifeval::["{cmdcli}" == "kubectl"]
+. Create certificates for IoT services. For testing purposes, you can create a self-signed certificates:
```
Comment (jenmalloy) [inline documentation/modules/proc-iot-installing-services.adoc]:
```suggestion
. Create certificates for IoT services. For testing purposes, you can create a self-signed certificate:
```

File: documentation/modules/proc-iot-installing-services.adoc [hunk 4]
```diff
@@ -13,11 +13,36 @@
 ----
 {cmdcli} apply -f install/preview-bundles/iot
 ----
+ifeval::["{cmdcli}" == "kubectl"]
+. Create certificates for IoT services. For testing purposes, you can create a self-signed certificate:
++
+[options="nowrap",subs="attributes"]
+----
+install/components/iot/examples/k8s-tls/create
```
Comment (jbtrystram) [inline documentation/modules/proc-iot-installing-services.adoc]: ./install/components/iot/examples/k8s-tls/create

**Pull request merged**